src/main/java/com/sparta/hirello/primary/column/entity/Columns.java

```diff
@@ -40,4 +33,13 @@ public class Columns {
 
     @OneToMany(mappedBy = "columns")
```

Comment:
```suggestion
    @OneToMany(mappedBy = "columns", cascade = CascadeType.ALL, orphanRemoval = true)
```
부모 엔티티와 생명주기를 같게 해야 합니다.